homwork.js

```diff
@@ -0,0 +1,102 @@
+////////////////////Q1///////////////////////////////
+function numberSum(N) {
+    var total = 0;
+      for(var i = 1; i <= N; i++){
+        total += i;
+      }
+      return total;
+  }
+
+  /////////////////////////Q2////////////////////////////////
+  function even_digits(num) {
+    var ctr = 0;
+    while (num) {
+      ctr += num % 2 === 0;
+      num = Math.floor(num / 10);
+    }
+    return ctr;
+  }
```

Comment:
The questions require a function that will take a number as the end of a range and sums all the even numbers from 1 to the end range number
so if we call `even_digits(5)` it should add 2 and 4 as they are the only even numbers between 1 and 5. The output would be 6.

Reply:
Yes, I need to update the answer with the code of the following snippet:
let x = 1;
let y = 2;
let sum = 0;
let limit = 10000;
let evensum = 2;

while ((x + y) < limit) {
    sum = x + y;
    x = y;
    y = sum;

    if (sum % 2 === 0) {
        evensum += sum;
    }
    console.log(evensum);
}